verifier/rest/rest.go

```diff
@@ -51,6 +69,9 @@ func NewClient(ctx context.Context, projectID string, region string, opts ...opt
 		return nil, fmt.Errorf("can't create ConfidentialComputing v1 API client: %w", err)
 	}
 
+	// Override the default retry CallOptions with specific retry policies.
```

Comment:
Is there a reference to the default retry CallOptions, why can't we just rely on the default retry options?

Reply:
The default retry options can be found in here [google3/google/cloud/confidentialcomputing/confidentialcomputing_v1_grpc_service_config.json](https://source.corp.google.com/piper///depot/google3/google/cloud/confidentialcomputing/confidentialcomputing_v1_grpc_service_config.json) and only retry on the code "UNAVAILABLE".

Reply:
cool, we may want to change the default json config later in the API, especially if we want to add more retry codes.

Add the OSS version here for reference:
https://github.com/googleapis/googleapis/blob/master/google/cloud/confidentialcomputing/v1/confidentialcomputing_v1_grpc_service_config.json